packages/dashboards/src/lib/components/widget/widget-header/widget-header.component.ts

```diff
@@ -67,7 +70,8 @@ export class WidgetHeaderComponent implements OnInit, OnDestroy, AfterViewInit {
     constructor(@Inject(PIZZAGNA_EVENT_BUS) private eventBus: EventBus<IEvent>,
                 public pizzagnaService: PizzagnaService,
                 public changeDetector: ChangeDetectorRef,
-                @Optional() private eventProxy: WidgetToDashboardEventProxyService) {
+                @Optional() private eventProxy: WidgetToDashboardEventProxyService,
+                @Optional() private linkProvider: HeaderLinkProvider) {
```

Comment:
I guess we need some sort of injection token here? This link provider is defined in the demo app.

Reply:
or, maybe just `IHeaderLinkProvider`?